src/compressed_tensors/base.py

```diff
@@ -14,4 +14,4 @@
 
 SPARSITY_CONFIG_NAME = "sparsity_config"
 QUANTIZATION_CONFIG_NAME = "quantization_config"
-COMPRESSION_CONFIG_NAME = "compression_config"
+COMPRESSION_CONFIG_NAME = "quantization_config"
```

Comment:
These are the same now? Could simply change the usage of COMPRESSION_CONFIG_NAME to QUANTIZATION_CONFIG_NAME where this causes an issue?

Reply:
This is a follow up to this PR: https://github.com/neuralmagic/compressed-tensors/pull/73. We are just changing the top level name in config.json from "compression_config" to "quantization_config" to be compatible with HfQuantizer. I'm leaving all 3 constants constants so that on the compressed-tensors side we can refer to the quantization and sparsity configs separately, while leaving COMPRESSION_CONFIG_NAME constant as just the name we use for the config.json.

Reply:
can we leave maybe a short comment that documents why we are being a bit confusing? (because of HF naming convention).